clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py

```diff
@@ -2,64 +2,197 @@
 
 """Module for converting FDG PET of ADNI."""
 
+from enum import Enum
+from functools import partial
+from os import PathLike
+from pathlib import Path
+from typing import List, Optional, Set, Tuple, Union
 
-def convert_adni_fdg_pet(
-    source_dir, csv_dir, dest_dir, conversion_dir, subjs_list=None, mod_to_update=False
+import pandas as pd
+
+
+class ADNIPreprocessingStep(Enum):
+    """ADNI preprocessing steps."""
+
+    STEP0 = "ADNI Brain PET: Raw FDG"
+    STEP1 = "Co-registered Dynamic"
+    STEP2 = "Co-registered, Averaged"
+    STEP3 = "Coreg, Avg, Standardized Image and Voxel Size"
+    STEP4 = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
+    STEP5 = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
```

Comment:
There are only 4 steps listed in the upstream [documentation](https://adni.loni.usc.edu/methods/pet-analysis-method/pet-analysis/).

I am guessing that there are different uniform resolutions (6 and 8 mm) and the docs was not updated appropriately. May I suggest the following alternative to avoid confusion:

```suggestion
    STEP4_6MM = "Coreg, Avg, Std Img and Voxel Size, Uniform 6mm Res"
    STEP4_8MM = "Coreg, Avg, Std Img and Voxel Size, Uniform Resolution"
```

WDYT?